config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
```

Comment:
Suggestion: Add routes of created models.

Reply:
I think routes make more sense when we have controllers in place. At the moment, they wouldn't do anything as the views are managed from the front-end and any route in place wouldn't return anything.